src/parameter_bridge.cpp

```diff
@@ -33,13 +33,13 @@
 
 int main(int argc, char * argv[])
 {
-  // ROS 1 node
-  ros::init(argc, argv, "ros_bridge");
-  ros::NodeHandle ros1_node;
-
   // ROS 2 node
   rclcpp::init(argc, argv);
   auto ros2_node = rclcpp::Node::make_shared("ros_bridge");
+  
+  // ROS 1 node
```

Comment:
perhaps we would want to add comments why this `rclcpp::init` needs to be called before `ros::init`.